torchvision/datasets/imagenet.py

```diff
@@ -21,6 +21,10 @@
 class ImageNet(ImageFolder):
     """`ImageNet <http://image-net.org/>`_ 2012 Classification Dataset.
 
+    Before using this class, it is required to download `ImageNet 2012` dataset from
+    `here<https://image-net.org/challenges/LSVRC/2012/2012-downloads.php>`_ and
```

Comment:
I think we need a space here: https://github.com/pytorch/vision/actions/runs/4667689696/jobs/8263804778?pr=7503#step:10:1218

```suggestion
    `here <https://image-net.org/challenges/LSVRC/2012/2012-downloads.php>`_ and
```